labgrid/driver/fastbootdriver.py

```diff
@@ -43,7 +43,13 @@ def on_deactivate(self):
     @Driver.check_active
     @step(title='call', args=['args'])
     def __call__(self, *args):
-        subprocess.check_call(self._get_fastboot_prefix() + list(args))
+        # unfortunately fastboot uses stderr for normal output
+        proc = subprocess.run(self._get_fastboot_prefix() + list(args), stdout=subprocess.PIPE,
+                              stderr=subprocess.STDOUT, check=True, universal_newlines=True)
+        stdout = proc.stdout.split('\n')
+        out_prefix = '(bootloader) '
+        # ignore lines with status and timing information, also remove message prefix
+        return list(line[len(out_prefix):] for line in stdout if line.startswith(out_prefix))
```

Comment:
This means that we no longer get output when using labgrid-client. Would keywork arg ``capture=True``work for your usecase? Or maybe even getvar(name), getvars()?

Reply:
Yes, agreed. I'll close this PR and open a new one adding `AndroidFastbootDriver.getvar(var)`, `AndroidFastbootDriver.oem_getenv(var)`.